### ISSUE TITLE
Hidden comments are not *obviously* hidden

### ISSUE THREAD

**Issue by andrew-vant** [Discordius/Lesswrong2#220]
I had a comment disappear after posting it, but it still appeared in my profile and I could reach it from there.

It turned out to have been hidden as part of the "showing 50/97 comments; show more" function. There is no visible indication that the comment is there (but hidden), but "show more" displays it along with everything else.

There should be such an indication, both on hidden subthreads and at the end of the comments if applicable.

**Comment by Discordius** [comment]
Yeah, strongly agree on this. I think we really want the page to present you various hints at various places that there are comments you haven't read yet. We probably could also optimize things to the point where we could practically always just load all the comments. I will want to experiment a bit with that. 

In the super short-term, I think we could optimize things so that we load 100 comments, instead of 50, initially, which will get rid of a good chunk of the problem.

**Comment by Raemon** [comment]
We have since dramatically increased the number of comments you can load (to 400). This issue still comes up on some old posts with huge numbers of comments but is much rarer.

**Comment by Raemon** [comment]
My preferred way to handle this is for, if there are more than the max-number of comments (400 I believe), a giant warning appears noting that not all comments have loaded. I'm fine with the warning being ugly for now since it shouldn't come up that often, and is more important that users know what's going on in those rare cases than that it still look pretty.
